cube/src/main/java/zone/cogni/asquare/cube/index/swap/JsonBuilder.java

```diff
@@ -0,0 +1,197 @@
+package zone.cogni.asquare.cube.index.swap;
+
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import javax.annotation.Nonnull;
+
+class JsonBuilder {
```

Comment:
seems reusable, can it be put in some json lib package?

Reply:
once we clean it, it definitely an option